src/test/ui/coherence/coherence-orphan.old.stderr

```diff
@@ -2,18 +2,23 @@ error[E0117]: only traits defined in the current crate can be implemented for ar
   --> $DIR/coherence-orphan.rs:13:1
    |
 LL | impl TheTrait<usize> for isize { }
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ impl doesn't use types inside crate
+   | ^^^^^---------------^^^^^-----
+   | |    |                   |
+   | |    |                   `isize` is not defined in the current crate
+   | |    `usize` is not defined in the current crate
```

Comment:
The span for `usize` is a bit odd: I would expect it to point to the generic parameter.

Reply:
We don't have a way to point at specific type params at the moment, but if be happy to add that in a follow up PR.

Reply:
Okay, sounds good to me :)